packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -52,64 +52,69 @@ interface CheckboxProps {
   onChange?(newValue: boolean): void;
 }
 
-export function Checkbox({
-  checked,
-  defaultChecked,
-  disabled,
-  label,
-  name,
-  value,
-  indeterminate = false,
-  description,
-  onChange,
-}: CheckboxProps) {
-  const wrapperClassName = classnames(
-    styles.wrapper,
-    disabled && styles.disabled,
-  );
-  const inputClassName = classnames(styles.input, {
-    [styles.indeterminate]: indeterminate,
-  });
-  const iconName = indeterminate ? "minus2" : "checkmark";
+export const Checkbox = React.forwardRef<HTMLDivElement, CheckboxProps>(
+  (
+    {
+      checked,
+      defaultChecked,
+      disabled,
+      label,
+      name,
+      value,
+      indeterminate = false,
+      description,
+      onChange,
+    },
+    ref,
+  ) => {
+    const wrapperClassName = classnames(
+      styles.wrapper,
+      disabled && styles.disabled,
+    );
+    const inputClassName = classnames(styles.input, {
+      [styles.indeterminate]: indeterminate,
+    });
+    const iconName = indeterminate ? "minus2" : "checkmark";
 
-  return (
-    <div className={styles.checkbox}>
-      <label className={wrapperClassName}>
-        <span className={styles.checkHolder}>
-          <input
-            type="checkbox"
-            checked={checked}
-            defaultChecked={defaultChecked}
-            className={inputClassName}
-            aria-label={label}
-            onChange={handleChange}
-            value={value}
-            name={name}
-            disabled={disabled}
-          />
-          <span className={styles.checkBox}>
-            <Icon name={iconName} size="small" color="white" />
+    return (
+      <div className={styles.checkbox} ref={ref}>
```

Comment:
The only other major change to the component is `ref={ref}`.  the rest are just spacing/tabbing.